server/base/src/main/java/org/apache/accumulo/server/init/Initialize.java

```diff
@@ -762,6 +764,17 @@ public void execute(final String[] args) {
       if (opts.resetSecurity) {
         AccumuloServerContext context = new AccumuloServerContext(new ServerConfigurationFactory(HdfsZooInstance.getInstance()));
         if (isInitialized(fs)) {
+          if (!opts.forceResetSecurity) {
+            String secret = acuConf.get(Property.INSTANCE_SECRET);
+            ConsoleReader c = getConsoleReader();
+            String userEnteredSecret = c.readLine("WARNING: This will remove all users from Accumulo! If you wish to proceed enter the instance secret: ", '*');
```

Comment:
> it could prompt for the instance name or ID

I like "instance name" instead of the secret. I do not believe the secret is self-identifying WRT the Github analogy you make.